docs/ceph-csi-upgrade.md

```diff
@@ -76,6 +78,11 @@ to upgrade from cephcsi v2.1 to v3.0
 
 ## Upgrading from v3.0 to v3.1
 
+Refer [upgrade-from-v3.0-v3.1](https://github.com/ceph/ceph-csi/blob/v3.1.0/docs/ceph-csi-upgrade.md)
```

Comment:
the link should be https://github.com/ceph/ceph-csi/blob/v3.1.2/docs/ceph-csi-upgrade.md v3.1.2 is the latest release we have

Reply:
fixed